_aggregations/metric/geocentroid.md

```diff
@@ -0,0 +1,256 @@
+---
+layout: default
+title: Geocentroid
+parent: Metric aggregations
+grand_parent: Aggregations
+nav_order: 45
+---
+
+# Geocentroid
+
+The `geo_centroid` aggregation in OpenSearch is a powerful tool that allows you to calculate the weighted geographic center or focal point of a set of spatial data points. This metric aggregation operates on `geo_point` fields and returns the centroid location as a latitude-longitude pair.
+
+## Using the aggregation
```

Comment:
Technical reviewer: Please confirm this example is relevant to an OpenSearch user and addresses the following user feedback: Undocumented behavior for when two indexes with overlapping field names are associated with the same alias. What happens when these are different types? There appears to be some kind of implicit casting that occurs. I tested the example using Dev Tools. If another example is more appropriate, please replace the draft example with your example. Thank you.

Reply:
LGTM